packages/katello/katello/katello.spec

```diff
@@ -51,8 +51,7 @@ Requires: pulp-puppet-tools
 Requires: pulp-selinux
 Requires: pulp-server
 Requires: python-pulp-streamer
-Requires: mongodb >= 2.4
-Requires: mongodb-server >= 2.4
+Requires: rh-mongodb34
```

Comment:
Shouldn't this be `rh-mongodb34-mongodb`, as `rh-mongodb34` is the meta package for the SCL itself?

Reply:
I think it can be -- from my testing both worked.

Reply:
looking here its https://www.softwarecollections.org/en/scls/rhscl/rh-mongodb34/

rh-mongodb34